modules/shortcodes/email-subscribe.php

```diff
@@ -0,0 +1,115 @@
+<?php
+/*
+ * WARNING: This file is distributed verbatim in Jetpack.
+ * There should be nothing WordPress.com specific in this file.
+ *
+ * @hide-in-jetpack
+ */
+class Jetpack_Email_Subscribe {
```

Comment:
I wonder if it would be worth describing things a bit here; what does the shortcode do exactly? Where do the users get subscribed? This way when we get back to this in 6 months we'll understand what this is used for.

Reply:
Description added.